cantools/database/can/message.py

```diff
@@ -55,7 +55,19 @@ def __init__(self,
         self._length = length
         self._signals = signals
         self._signals.sort(key=start_bit)
-        self._comment = comment
+
+        # if the 'comments' argument is a string, we assume that is an
+        # english comment. this is slightly hacky because the
+        # function's behavior depends on the type of the passed
+        # argument, but it is quite convenient...
+        self._comments = {}
+        if isinstance(comments, str):
+            # use the first comment in the dictionary as "The" comment
+            self._comments = { "EN": comments }
+        elif comments is not None:
+            # assume that we have a multi-lingual dictionary
+            self._comments = comments
```

Comment:
Replace `elif comments is not None:` with `else:` as `self._comments` should be `None` if there are no comments.

Reply:
IMO this would be a slightly bad idea  because the code could no longer rely on the fact that the `_comments` attribute always is a dictionary. Shall I still change it?

Reply:
Well, the docstrings for both the `comment` and the `comments` properties says that it is `None` if there are no comments available. And `self._comments` is return from one of them.

The `comment` property should at least continue to return `None` if there is no comment. The `comments` property could return I dictionary I guess, and in that case the docstring should be updated.

Reply:
good point. I've changed it in the latest revision.